lib/vanagon/common/pathname.rb

```diff
@@ -28,7 +30,7 @@ class Pathname
       #   and exposed through the {#configfile?} method.
       # @return [Vanagon::Common::Pathname] Returns a new Pathname instance.
       def initialize(path, mode: nil, owner: nil, group: nil, config: false)
-        @path = File.expand_path(path)
+        @path = ::Pathname.new(path).cleanpath.to_s
```

Comment:
It looks like we should probably use `#realpath` instead of `#cleanpath` as `#cleanpath` runs the risk of containing `..`.

http://ruby-doc.org/stdlib-2.1.0/libdoc/pathname/rdoc/Pathname.html#method-i-cleanpath
http://ruby-doc.org/stdlib-2.1.0/libdoc/pathname/rdoc/Pathname.html#method-i-realpath

Yes, I know I'm the one who said "use `#cleanpath`" and I am sorry.